commonknowledge/meep-intelligence-hub
# feat: add 2024 constituencies to import_areas command

## Description
Adds the 2024 westminster constituencies to the `import_areas` command.
This **is not** how Local Intelligence Hub imports them, which I have not yet found.
However, it does make the "Future Constituencies" map work correctly:

<img width="1201" alt="image" src="https://github.com/commonknowledge/meep-intelligence-hub/assets/12935136/d65a68db-3814-40e2-9517-acf022b2c467">


## Motivation and Context
Required for the TCC hub map.

## How Can It Be Tested?
Run `python manage.py import_areas` and verify the map here: http://localhost:8000/explore/?area_type=WMC23 looks like the above screenshot.

## Types of changes
